week2/Week_2 (DraftForClass).ipynb

```diff
@@ -0,0 +1,234 @@
+{
```

Comment:
<p>This is a great question.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-wildbillcolorado/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>